modules/test/playwright/tests/client-extension-web/js.spec.ts

```diff
@@ -321,3 +326,83 @@ test('GlobalJS client extension with async and defer attributes set to false and
 		],
 	});
 });
+
+test('GlobalJS client extension can be created with name translations while having a language configuration for the site settings', async ({
```

Comment:
@thiago-buarqque @dsanz The code looks good, and we can ship it as it is, but a quick question: should this test be in `globalJS.spec.ts`, as well as several others in the file starting with `GlobalJS`? Or should the content of `globalJS.spec.ts` be merged into `js.spec.ts`? What's the difference between the two? There is only one `JS` CX in UI.

Reply:
I think it makes sense to merge these files. I'm not sure what was the purpose of adding the globaljs one. I'm going to do that and forward to Brian

Reply:
Found the root cause. Ivan [added tests for the globaljs sample](https://github.com/liferay/liferay-portal/commit/97e5f5e308f31aa4cd304b684c6b99bb36298b47). We had a file called globaljs.spec.ts, but it was renamed to the js.spec.ts we have today ([commits](https://github.com/liferay/liferay-portal/commits/97e5f5e308f31aa4cd304b684c6b99bb36298b47/modules/test/playwright/tests/client-extension-web/js.spec.ts)).